test/end-to-end/pages/createBlueprint.js

```diff
@@ -41,7 +41,7 @@ module.exports = class CreateBlueprintPage extends BlueprintPage {
 
     // Help-block error message
     this.varHelpBlockMsg = 'A blueprint name is required.';
-    this.spanHelpBlockMsg = `${this.dialogRootElement} span[class="help-block"]`;
+    this.spanHelpBlockMsg = `${this.dialogRootElement} #help-block-msg`;
```

Comment:
After making the updates in my previous comment, you'll need to replace the `id` selector with the `data-id` attribute selector.